readme.md

```diff
@@ -230,6 +277,7 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+Instructor.find(name), function () {
+  return (wishlist items)
+};
 ```
```

Comment:
There are a few different ways to do this, one of them being the below. I suggest reviewing Mongoose syntax and finding equivalencies between it and Active Record.

```js
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```